schemas/com.olark/transcript/jsonschema/1-0-0

```diff
@@ -0,0 +1,202 @@
+        },
+        "additionalProperties": false
+      }
+    },
+    "groups": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "kind": {
+            "type": "string",
+            "maxLength": 128
+          },
+          "name": {
+            "type": "string",
+            "maxLength": 128
+          },
+          "id": {
+            "type": "string",
+            "maxLength": 128
+          }
+        },
+        "additionalProperties": false
+      }
+    },
+    "visitor": {
+      "type": "object",
+      "properties": {
+        "organization": {
+          "type": "string",
+          "maxLength": 128
+        },
+        "city": {
+          "type": "string",
+          "maxLength": 128
+        },
+        "ip": {
+          "type": "string",
+          "format": "ipv4"
+        },
+        "phoneNumber": {
+          "type": "string",
+          "maxLength": 64
+        },
+        "browser": {
+          "type": "string",
+          "maxLength": 128
+        },
+        "fullName": {
+          "type": "string",
+          "maxLength": 256
+        },
+        "emailAddress": {
+          "type": "string",
+          "maxLength": 512
+        },
+        "country": {
+          "type": "string",
+          "maxLength": 128
+        },
+        "operatingSystem": {
+          "type": "string",
+          "maxLength": 128
+        },
+        "id": {
+          "type": "string",
+          "maxLength": 64
+        },
+        "countryCode": {
+          "type": "string",
+          "maxLength": 8
+        },
+        "region": {
+          "type": "string",
+          "maxLength": 8
+        },
+        "conversationBeginPage": {
+          "type": "string",
+          "format": "uri",
+          "maxLength": 8192
+        },
+        "chat_feedback": {
+          "type": "object",
+          "properties": {
+            "knowledge": {
+              "type": "integer",
+              "maximum": 32767,
+              "minimum": 0
+            },
+            "friendliness": {
+              "type": "integer",
+              "maximum": 32767,
+              "minimum": 0
+            },
+            "responsiveness": {
+              "type": "integer",
+              "maximum": 32767,
+              "minimum": 0
+            },
+            "overall_chat": {
+              "type": "integer",
+              "maximum": 32767,
+              "minimum": 0
+            },
+            "comments": {
+              "type": "string"
+            }
+          },
+          "additionalProperties": false
+        },
+        "kind": {
+          "type": "string",
+          "maxLength": 64
+        }
+      },
+      "additionalProperties": true
+    },
+    "id": {
+      "type": "string",
+      "maxLength": 128
+    },
+    "kind": {
+      "type": "string",
+      "maxLength": 64
+    },
+    "operators": {
+      "type": "object",
+      "properties": {
+        ".*": {
```

Comment:
Maybe you wanted to use `patternProperties` here? I don't think this is valid schema.

Reply:
Yes, thank you.